test/config/bundles.js

```diff
@@ -0,0 +1,38 @@
+import chai from 'chai';
```

Comment:
As @pgoetze set it up that way in another PR, because of a standardising the test setup across projects (as discussed with @artibella)